.crews.php

```diff
@@ -47,9 +47,16 @@ function clearCrew ($connection, $signature, $id, $position, $crewid) {
     modifyCrewAssignment($connection, $signature, $id, $position, $crewid, 'clear');
 }
 
-function determineEligibility ($member, $pos, $i, $ontoday, $onthisweek, $ccton, $dton, $atton, $obson, $y, $m, $d) {
+function determineEligibility ($member, $pos, $i, $ontoday, $onthisweek, $ccton, $dton, $atton, $obson, $y, $m, $d, $dob) {
+    $dob = explode("-", $dob);
+    $age = (date("md", date("U", mktime(0, 0, 0, $dob[0], $dob[1], $dob[2]))) > date("md")
+      ? ((date("Y") - $dob[2]) - 1)
+      : (date("Y") - $dob[2]));
```

Comment:
The format of the `dob` field is `YYYY-MM-DD` so your array keys need to be reversed (replace `0` with `2` and vice versa).